src/controller/python/chip/storage/__init__.py

```diff
@@ -151,7 +153,7 @@ def SetSdkKey(self, key: str, value: bytes):
             raise ValueError("Invalid Key")
 
         if (value is None):
-            del(self.jsonData['sdk-config'][key])
+            self.jsonData['sdk-config'][key] = None
```

Comment:
I think you have to set to "" otherwise it may not read-back.

Does this pass the test of writing a 0-length key and reading it back as length-length?